docs/README.md

```diff
@@ -13,6 +13,11 @@ pip install sphinx_rtd_theme
 make html
 ```
 
+Shortcut to build the docs (with env variables for active-learning tutorial) use:
+```
+LIGHTLY_SERVER_LOCATION='https://api.lightly.ai' TOKEN='YOUR_TOKEN' AL_TUTORIAL_DATASET_ID='YOUR_DATASET_ID' make html && python -m http.server 1234 -d build/html
```

Comment:
do you always want to spcify the LIGHLLY_SERVER_LOCATION?

Reply:
This is what I use to run it. You can also export the variable. I prefer setting it every time so I don't accidentally have all my lightly stuff connecting to the dev API whereas I want to run on the prod one :)